src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -197,6 +216,17 @@ public SAMFileHeader getHeaderForReads() {
     }
 
     /**
+     * Returns the read filter (simple or composite) that will be applied to the reads returned from {@link #getReads}
+     * The default implementation uses the {@link org.broadinstitute.hellbender.engine.filters.WellformedReadFilter} filter with all default options.
+     *
+     * Subclasses can extend to provide their own filters (ie., override and optionally call super).
+     * Multiple filters can be composed by using {@link org.broadinstitute.hellbender.engine.filters.ReadFilter} composition methods.
+     */
+    public CountingReadFilter makeReadFilter() {
```

Comment:
Should we just use a regular read filter instead of a counting one?  It seems like minor but unnecessary overhead since the counts won't be used in any way?